src/PactNet.Native/Verifier/NativePactBrokerOptions.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using PactNet.Verifier;
+
+namespace PactNet.Native.Verifier
+{
+    /// <summary>
+    /// Native pact broker options
+    /// </summary>
+    internal class NativePactBrokerOptions : IPactBrokerOptions
+    {
+        private readonly IDictionary<string, string> verifierArgs;
+
+        /// <summary>
+        /// Initialises a new instance of the <see cref="NativePactBrokerOptions"/> class.
+        /// </summary>
+        /// <param name="verifierArgs">Pact verifier args</param>
+        public NativePactBrokerOptions(IDictionary<string, string> verifierArgs)
+        {
+            this.verifierArgs = verifierArgs;
+        }
+
+        /// <summary>
+        /// Use Basic authentication with the Pact Broker
+        /// </summary>
+        /// <param name="username">Pact broker username</param>
+        /// <param name="password">Pact broker password</param>
+        /// <returns>Fluent builder</returns>
+        public IPactBrokerOptions BasicAuthentication(string username, string password)
+        {
+            this.verifierArgs.Add("--user",     username ?? throw new ArgumentNullException(nameof(username)));
```

Comment:
I wouldn't add extra space for lining up variable. I know it's for lisiblity but on different IDE, it can look different.

Reply:
Yeah fair. I think the IDE did it automatically because I never do that either.